app/controllers/code_schools_controller.rb

```diff
@@ -7,4 +7,20 @@ def index
     @code_schools_by_state = code_schools.by_state
     @country = Country.named('United States')
   end
+
+  def map
+    code_schools = CodeSchools.new.all
+    @lat_longs = []
+
+    puts "first code school #{code_schools[0]}"
+
+    code_schools.each do |hash| 
+      if hash.key?("latitude") && hash.key?("longitude")
+        @lat_longs << [hash["longitude"], hash["latitude"]]
+      else 
+        puts "The latitude and longitude are not present"
```

Comment:
Why the `puts` statements here and on line 15?  Is that for your debugging an issue?  

If the goal is for the content in those lines to persist in some way, you'll want to use a `Rails.logger.info "some message content"` in place of those lines.  That way, the content will print out to the server logs.  

Or alternatively, is your intent to provide the user with feedback?  If yes, you'd want to use a `flash[:error]` message.

That said, was is the goal here?